bin/deploy.sh

```diff
@@ -24,20 +24,24 @@ export KUBE_NAMESPACE=$1
 export DRONE_SOURCE_BRANCH=$(echo $DRONE_SOURCE_BRANCH | tr '[:upper:]' '[:lower:]' | tr '/' '-')
 
 if [[ ${KUBE_NAMESPACE} == ${BRANCH_ENV} ]]; then
+  export DISCOVERY_URL="https://acp-sso.notprod.acp.homeoffice.gov.uk/realms/rotm-dev"
   $kd -f kube/file-vault/file-vault-ingress.yml # deploy ingress first so file-vault can use its tls-secret in its keycloak certs
   $kd -f kube/configmaps -f kube/certs
   $kd -f kube/redis -f kube/file-vault -f kube/app
 elif [[ ${KUBE_NAMESPACE} == ${UAT_ENV} ]]; then
+  export DISCOVERY_URL="https://acp-sso.notprod.acp.homeoffice.gov.uk/realms/rotm-dev"
   $kd -f kube/file-vault/file-vault-ingress.yml
   $kd -f kube/configmaps/configmap.yml -f kube/app/service.yml
   $kd -f kube/app/ingress-internal.yml -f kube/app/networkpolicy-internal.yml
   $kd -f kube/redis -f kube/file-vault -f kube/app/deployment.yml
 elif [[ ${KUBE_NAMESPACE} == ${STG_ENV} ]]; then
+  export DISCOVERY_URL="https://acp-sso.notprod.acp.homeoffice.gov.uk/realms/rotm-dev"
   $kd -f kube/file-vault/file-vault-ingress.yml
   $kd -f kube/configmaps/configmap.yml  -f kube/app/service.yml
   $kd -f kube/app/ingress-internal.yml -f kube/app/networkpolicy-internal.yml
   $kd -f kube/redis -f kube/file-vault -f kube/app/deployment.yml
 elif [[ ${KUBE_NAMESPACE} == ${PROD_ENV} ]]; then
+  export DISCOVERY_URL="https://sso.digital.homeoffice.gov.uk/auth/realms/rotm"
```

Comment:
It's a good idea to do your dev/test like this
but for the final solution we are going to move this to right when ACP have created a prod version for keycloak v22? [hof-config](https://github.com/UKHomeOfficeForms/hof-services-config)

Reply:
maybe add a comment

Reply:
> It's a good idea to do your dev/test like this but for the final solution we are going to move this to right when ACP have created a prod version for keycloak v22? [hof-config](https://github.com/UKHomeOfficeForms/hof-services-config)

Yes there will be a new discovery URL for prod when acp upgrade keycloak in prod. I will add a comment in about this. Thanks